### ISSUE TITLE
Does it support int8 quantization?

### ISSUE THREAD

**Issue by O-O1024** [bes-dev/stable_diffusion.openvino#127]
if it supports int8 quantization, the model size can be reduced 3/4. May be faster.

**Comment by RedAndr** [comment]
Yes, it is twice as small as the 16-bit model, but the quality is terrible, I must admit.
